DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1

```diff
@@ -0,0 +1,264 @@
+$currentPath = Split-Path -Parent $MyInvocation.MyCommand.Path
```

Comment:
Please add a header for this resource psm1 with resource description and what it is for.